### PR TITLE
pie-microsite@v1.45.1 - Address design reviews for design tokens/typography pages

### PR DESCRIPTION
**Pull request by raoufswe**
This PR is a follow-up to my previous work on typography and design token pages to address the designer's notes.

### Fixed
- Align the notification icon to the top
- Bottom margin of prism code snippets to be 24px
- Lists to have a 32px bottom margin if it's followed by a heading
- h2 to have a top margin of 56px if followed by a paragraph
- Headings hierarchy of typography code snippets page

### CODE REVIEW

**Comment by jamieomaguire** [comment]
Screenshot 2023-01-25 at 09 05 20

@raoufswe this appears to break mobile views by adding horizontal scrolling

**Comment by jamieomaguire** [comment]
Hmm, upon retrying it could be that my chrome messed up - sorry if that's the case!

**Review by jamieomaguire — DISMISSED** [review]

File: apps/pie-microsite/src/assets/styles/plugins/_prism.scss [hunk 1]
```diff
@@ -121,7 +121,7 @@ div.code-toolbar {
   position: relative;
   max-width: 1025px;
   // overflow: hidden;
-  margin: 25px 0px 40px;
+  margin: 24px 0px;
```
Comment (jamieomaguire) [inline apps/pie-microsite/src/assets/styles/plugins/_prism.scss:124]: can you use `f.spacing(e)` here please?

Comment (jamieomaguire) [reply apps/pie-microsite/src/assets/styles/plugins/_prism.scss:124]: for reference: https://github.com/justeattakeaway/fozzie-components/blob/7fd01766b3126b30b1ab704c131b8ac767f882ea/packages/tools/fozzie/src/scss/settings/_variables.scss#L94

**Review by FayeCarter — APPROVED** [review]

**Review by fernandofranca — APPROVED** [review]

**Pull request merged**